index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+<h1 id= "title">Hello world!</h1> 
+<form id="survey-form">
+  <label type="text" id="name-label">Name</label>
+  <input type="text" id="name" placeholder="Name" required><br>
+  <label type="email" id="email-label">Email</label>
+  <input type ="email" id="email" placeholder="email" required><br>
+  <label type="number" id="number-label">Age</label>
+  <input type="number" id="number" min="18" max="75" placeholder="Age"><br>
+  <select id="dropdown">
+    <option value ="Alexandre">Alexandre</option>
+    <option value="Cesar">Cesar</option>
+  </select><br>
+  <input type="radio" name="name" value="alex"><br>
+  <input type="radio" name="name" value="cesar"><br>
+  <input type="checkbox" value="name"><br>
+  <input type="checkbox" value="name"><br>
+  <textarea id="comments">place your comemnte</textarea><br>
+  <button id="submit" type="submit">Submit</button><br>
+</form>
+  <body>
+    <p id="description">This is a survey</p>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
kindly recommend that don't put comments in your code if its work with a function